src/main/java/com/alkemy/ong/infrastructure/config/spring/exception/RestExceptionHandler.java

```diff
@@ -0,0 +1,42 @@
+package com.alkemy.ong.infrastructure.config.spring.exception;
+
+import com.alkemy.ong.application.exception.ObjectNotFound;
+import com.alkemy.ong.infrastructure.rest.response.ErrorResponse;
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+
+@ControllerAdvice
+public class RestExceptionHandler {
+
+  private static final String OBJECT_NOT_FOUND = "Object not found in database.";
+
+  @ExceptionHandler(value = ObjectNotFound.class)
+  protected ResponseEntity<ErrorResponse> handleObjectNotFound(ObjectNotFound e) {
+    ErrorResponse errorResponse = new ErrorResponse(HttpStatus.NOT_FOUND.toString(),
+        OBJECT_NOT_FOUND, e.getMessage());
+    return new ResponseEntity<>(errorResponse, HttpStatus.NOT_FOUND);
+  }
+
+  @ExceptionHandler(value = MethodArgumentNotValidException.class)
+  protected ResponseEntity<Object> handleMethodArgumentNotValid(MethodArgumentNotValidException ex,
+      HttpStatus status) {
+
+    List<String> errors = new ArrayList<>();
+
+    ex.getBindingResult().getFieldErrors().forEach(fieldError -> errors.add(
+        String.format("%s : %s", fieldError.getField(), fieldError.getDefaultMessage())));
+
+    ex.getBindingResult().getGlobalErrors().forEach(objectError -> errors.add(
+        String.format("%s : %s", objectError.getObjectName(), objectError.getDefaultMessage())));
+
+    ErrorResponse errorResponse = new ErrorResponse(status.toString(), ex.getLocalizedMessage(),
```

Comment:
ex.getLocalizedMessage() no es un human readable message. Ese mensaje es algo como "Validation failed for argument {....} in {package} . {fieldErrors list}"